scm/src/scm_input.F90

```diff
@@ -922,82 +932,225 @@ subroutine get_case_init_DEPHY(scm_state, scm_input)
+  real(kind=dp), allocatable  :: input_sncovr_ice(:)      !<
+  real(kind=dp), allocatable  :: input_sfalb_lnd(:)       !<
+  real(kind=dp), allocatable  :: input_sfalb_lnd_bck(:)   !<
+  real(kind=dp), allocatable  :: input_sfalb_ice(:)       !<
+  real(kind=dp), allocatable  :: input_emis_ice(:)        !<
   
   ! forcing variables
-  real(kind=sp), allocatable :: input_force_pres_surf(:,:,:) !< forcing surface pressure (Pa)
-  real(kind=sp), allocatable :: input_force_height(:,:,:,:) !< forcing height levels (m)
-  real(kind=sp), allocatable :: input_force_pres(:,:,:,:) !< forcing pressure levels (Pa)
-  real(kind=sp), allocatable :: input_force_u_g(:,:,:,:) !< forcing 2D geostrophic east-west wind (m s^-1)
-  real(kind=sp), allocatable :: input_force_v_g(:,:,:,:) !< forcing 2D geostrophic north-south wind (m s^-1)
-  real(kind=sp), allocatable :: input_force_w(:,:,:,:)
-  real(kind=sp), allocatable :: input_force_omega(:,:,:,:)
-  real(kind=sp), allocatable :: input_force_u_adv(:,:,:,:)
-  real(kind=sp), allocatable :: input_force_v_adv(:,:,:,:)
-  real(kind=sp), allocatable :: input_force_temp_adv(:,:,:,:)
-  real(kind=sp), allocatable :: input_force_theta_adv(:,:,:,:)
-  real(kind=sp), allocatable :: input_force_thetal_adv(:,:,:,:)
-  real(kind=sp), allocatable :: input_force_qt_adv(:,:,:,:)
-  real(kind=sp), allocatable :: input_force_qv_adv(:,:,:,:)
-  real(kind=sp), allocatable :: input_force_rt_adv(:,:,:,:)
-  real(kind=sp), allocatable :: input_force_rv_adv(:,:,:,:)
-  real(kind=sp), allocatable :: input_force_temp_rad(:,:,:,:)
-  real(kind=sp), allocatable :: input_force_theta_rad(:,:,:,:)
-  real(kind=sp), allocatable :: input_force_thetal_rad(:,:,:,:)
-  real(kind=sp), allocatable :: input_force_sfc_sens_flx(:,:,:)
-  real(kind=sp), allocatable :: input_force_sfc_lat_flx(:,:,:)
-  real(kind=sp), allocatable :: input_force_wpthetap(:,:,:)
-  real(kind=sp), allocatable :: input_force_wpqvp(:,:,:)
-  real(kind=sp), allocatable :: input_force_wpqtp(:,:,:)
-  real(kind=sp), allocatable :: input_force_wprvp(:,:,:)
-  real(kind=sp), allocatable :: input_force_wprtp(:,:,:)
-  real(kind=sp), allocatable :: input_force_ts(:,:,:)
-  real(kind=sp), allocatable :: input_force_ustar(:,:,:)
-  real(kind=sp), allocatable :: input_force_temp_nudging(:,:,:,:)
-  real(kind=sp), allocatable :: input_force_theta_nudging(:,:,:,:)
-  real(kind=sp), allocatable :: input_force_thetal_nudging(:,:,:,:)
-  real(kind=sp), allocatable :: input_force_qv_nudging(:,:,:,:)
-  real(kind=sp), allocatable :: input_force_qt_nudging(:,:,:,:)
-  real(kind=sp), allocatable :: input_force_rv_nudging(:,:,:,:)
-  real(kind=sp), allocatable :: input_force_rt_nudging(:,:,:,:)
-  real(kind=sp), allocatable :: input_force_u_nudging(:,:,:,:)
-  real(kind=sp), allocatable :: input_force_v_nudging(:,:,:,:)
+  real(kind=sp), allocatable :: input_force_pres_surf(:) !< forcing surface pressure (Pa)
```

Comment:
In the description, you note a change in dimensions in the DEPHY v1 file, but I must have interpreted your description wrong. You said "from DEPHY v0(?) -> v1 there was a change in the format from [x, y, z] -> [column, z]". This looks like there is every only data for 1 column and the dimensions for 1D arrays are [time], and the dimensions for 2D arrays are [z, time]. Is that right?